templates/UNRELEASED_CHANGES_unified.md.tmpl

```diff
@@ -11,6 +11,9 @@ This file documents all changes that have not been released yet
 ## Unreleased Components
 
 These are the component versions that have yet not been included in the Conjur OSS Suite:
+{{- range .ComponentsByCategory }}
+
+### {{ .CategoryName }}
```

Comment:
Curious question - what happens if a category is empty in all these templates?

Reply:
I don't think this is relevant - we determine what's set in the suite.yml, and we have everything mapped to categories now

Reply:
If there are no unreleased changes, the component may be missing from the category (and the category could end up empty) I think

Reply:
I actually think there are bugs here that I'm not sure my PR addresses - look at the current wiki, it shows .net lib as 1.4.0 in unreleased when that version is in suite.yml and also in the latest `releases/` suite.yml

so I agree this will need to be addressed, but I think that should be a follow-on issue that addresses the larger issues in this report

Reply:
Added #154 to track needed improvements to the unreleased wiki page, including categories with no unreleased components (though with the stuff on master also included here, that'll probably be rare)